### ISSUE TITLE
Unable to build jazzy docset

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-navigation-ios#2363]
scripts/document.sh is failing because CocoaPods is unable to build MapboxNavigation-Documentation.podspec using Xcode 11.4:

```
Running xcodebuild
Could not successfully run `xcodebuild`.
Please check the build arguments.
Saved `xcodebuild` log file: /var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/xcodebuild-BA3B43B6-1171-4F10-9517-38038FCBAD4B.log
Failed to generate documentation
Traceback (most recent call last):
	12: from /usr/local/bin/jazzy:23:in `<main>'
	11: from /usr/local/bin/jazzy:23:in `load'
	10: from /usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/bin/jazzy:15:in `<top (required)>'
	 9: from /usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/lib/jazzy/doc_builder.rb:74:in `build'
	 8: from /usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/lib/jazzy/podspec_documenter.rb:18:in `sourcekitten_output'
	 7: from /usr/local/lib/ruby/gems/2.6.0/gems/cocoapods-1.9.1/lib/cocoapods/config.rb:41:in `with_changes'
	 6: from /usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/lib/jazzy/podspec_documenter.rb:24:in `block in sourcekitten_output'
	 5: from /usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/lib/jazzy/podspec_documenter.rb:24:in `chdir'
	 4: from /usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/lib/jazzy/podspec_documenter.rb:29:in `block (2 levels) in sourcekitten_output'
	 3: from /usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/lib/jazzy/podspec_documenter.rb:29:in `map'
	 2: from /usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/lib/jazzy/podspec_documenter.rb:31:in `block (3 levels) in sourcekitten_output'
	 1: from /usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/lib/jazzy/sourcekitten.rb:266:in `run_sourcekitten'
/usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/lib/jazzy/executable.rb:36:in `execute_command': /usr/local/lib/ruby/gems/2.6.0/gems/jazzy-0.13.2/bin/sourcekitten ["doc", "--module-name", "MapboxNavigation", "--", "-target", "MapboxNavigation-Documentation"] (RuntimeError)

Running xcodebuild

Could not successfully run `xcodebuild`.

Please check the build arguments.

Saved `xcodebuild` log file: /var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/xcodebuild-BA3B43B6-1171-4F10-9517-38038FCBAD4B.log

Failed to generate documentation
```

```
error: Multiple commands produce '/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/jazzy20200417-99425-1lnlz2qMapboxNavigation-Documentation/build/Release-iphoneos/MapboxNavigation-Documentation/MapboxNavigation.framework/Info.plist':
1) Target 'MapboxNavigation-Documentation' (project 'Pods') has copy command from '/path/to/MapboxCoreNavigation/Info.plist' to '/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/jazzy20200417-99425-1lnlz2qMapboxNavigation-Documentation/build/Release-iphoneos/MapboxNavigation-Documentation/MapboxNavigation.framework/Info.plist'
2) Target 'MapboxNavigation-Documentation' (project 'Pods') has copy command from '/path/to/MapboxNavigation/Info.plist' to '/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/jazzy20200417-99425-1lnlz2qMapboxNavigation-Documentation/build/Release-iphoneos/MapboxNavigation-Documentation/MapboxNavigation.framework/Info.plist'
3) Target 'MapboxNavigation-Documentation' (project 'Pods') has process command with output '/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/jazzy20200417-99425-1lnlz2qMapboxNavigation-Documentation/build/Release-iphoneos/MapboxNavigation-Documentation/MapboxNavigation.framework/Info.plist'
…
/path/to/MapboxNavigation/BottomBannerViewController.swift:2:8: error: no such module 'MapboxCoreNavigation'
import MapboxCoreNavigation
       ^
```

This is blocking us from publishing the v0.39.1 documentation on mapbox.com.

/cc @mapbox/navigation-ios

**Comment by 1ec5** [comment]
This is the jazzy invocation that fails:

https://github.com/mapbox/mapbox-navigation-ios/blob/c83f0448aabb169cd90a88e2bf159df6df6174b4/scripts/document.sh#L53-L64

The `--podspec` argument refers to https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation-Documentation.podspec, which fakes a target that contains both the MapboxCoreNavigation and MapboxNavigation sources and resources.

I suspect the error above about the duplicate Info.plist copy commands is relevant. It shouldn’t be necessary to explicitly copy Info.plists; not sure how that started happening.

**Comment by 1ec5** [comment]
Fixed in #2380 and cherry-picked to the release-v0.39 branch in 1d6cf6d4cad991500b40b320ad5dbfdc467b37a8, though that branch also required c713c4ee45a2302975ee590ffa55005a2e289aac to avoid MapboxSpeech v0.3.0.
